docs/source/quickstart-docker.rst

```diff
@@ -1,49 +1,60 @@
-Using Docker
-============
+Install JupyterHub with Docker
+============================================
 
-.. important::
+The JupyterHub `docker image <https://hub.docker.com/r/jupyterhub/jupyterhub/>`_ is the fastest way to set up Jupyterhub in your local development environment.
 
-   We highly recommend following the `Zero to JupyterHub`_ tutorial for
-   installing JupyterHub.
+The ``JupyterHub`` docker image runs the Hub service only. It does not provide other Jupyter components such as Notebook installation, which are needed by the single-user servers.
+To run the single-user servers, which may be on the same system as the Hub or not, Jupyter Notebook version 4 or greater must be installed.
 
-Alternate installation using Docker
------------------------------------
+.. important::
+    We strongly recommend that you follow the `Zero to JupyterHub`_ tutorial to
+    install JupyterHub.
+   
+   
+Prerequisites
+-------------------------------
+You should have `Docker Desktop`_ installed on a Linux/Unix based system.
```

Comment:
It doesn't need to specify the platform - this should work anywhere docker works